cds/modules/fixtures/data/pages/guides/search.html

```diff
@@ -0,0 +1,398 @@
+
+<p>Note: <code>^</code> is followed by a positive floating point number specifying the boost value compared to the default 1.
+
+<h4 id="fuzziness">Fuzziness</h4>
+
+<p>You can search for terms similar to but not exactly like your search term using the fuzzy operator <code>~</code>.</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=drak~"><code>drak~</code></a></p>
+
+<p>Results will match records with terms similar to <code>drak</code> which would e.g. also match <code>dark</code>.</p>
+
+<h4 id="proximitysearches">Proximity searches</h4>
+
+<p>A phrase search like <code>"president visit"</code> by default expect all terms in exactly the same order, and thus for instance would not match a record containing the phrase <em>"Visit of his excellency the President of..."</em>. A proximity search allows that the terms are not in the exact order and may include other terms inbetween. The degree of flexiblity is specified by an integer afterwards:</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=%22president%20visit%22~10"><code>"president visit"~10</code></a></p>
+
+<h4 id="wildcards">Wildcards</h4>
+
+<p>You can use wildcards in search terms to replace a single character (using <code>?</code> operator) or zero or more characters (using <code>*</code> operator).</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=title.title:(crab%20cavit*)"><code>title.title:(crab cavit*)</code></a></p>
+
+The result will match videos with title referring to cavities also in languages other than english.
+
+<p>Wildcard searches can be slow and should normally be avoided if possible.</p>
+
+<hr />
+
+<h2 id="fieldsreference">Fields reference</h2>
+
+<p>The table below lists the data type of each field. Below is a quick description
+of what each data type means and what is possible.</p>
+
+<ul>
+<li><strong>string</strong>: Field does not require exact match (example field: <code>title.title</code>).</li>
+
+<!--
+
+<li><strong>string (exact)</strong>: Field requires exact match (example field: <code>doi</code>).</li>
+
+<li><strong>CV</strong>: Controlled vocabularies are similar to "string (exact)" but takes only a specific list of values.</li>
+
+<li><strong>integer</strong>: A number (range queries possible).</li>
+-->
+<li><strong>datetime</strong>: An ISO8601 date or datetime, e.g. <code>2017-12-31</code> (range queries possible).</li>
+
+<li><strong>boolean</strong>: <code>true</code> or <code>false</code>.</li>
+</ul>
+
+<p>
+<table><thead><tr>
+<th align="left">Field name</th>
+<th align="left">Type</th>
+<th align="left">Notes</th></tr></thead>
+<tbody>
+<tr>
+  <td><code>category</code></td>
+  <td>string</td>
+  <td>Category of the project</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>contributors.\*</code></td>
```

Comment:
I think we should say somewhere that `\*` means searching in the subfields of a field